### PR TITLE
Create 2015-06-23-contour-colorbar-pos

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp

### CODE REVIEW

File: _posts/python/contour/2015-06-23-contour-colorbar-pos [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+data = Data([
+    Contour(
+        z=[[10, 10.625, 12.5, 15.625, 20],
+           [5.625, 6.25, 8.125, 11.25, 15.625],
+           [2.5, 3.125, 5., 8.125, 12.5],
+           [0.625, 1.25, 3.125, 6.25, 10.625],
+           [0, 0.625, 2.5, 5.625, 10]],
+        colorbar=ColorBar(
+       		x=1,
```
Comment (chriddyp) [inline _posts/python/contour/2015-06-23-contour-colorbar-pos]: @aneda - could we move this to the left-side instead? This colorer position looks too much like the rest of the examples (right-side)

**Comment by aneda** [comment]
@chriddyp is this better now?

**Comment by chriddyp** [comment]
@aneda cool. let's move the ticks to the left of the bar though, so that they aren't overlapping with the yaxis ticks

**Comment by aneda** [comment]
@chriddyp can we do that?  I can't figure it out

**Comment by chriddyp** [comment]
nvm, you're totally right, not possible yet

**Pull request merged**